heartbeat/_meta/beat.yml

```diff
@@ -9,7 +9,17 @@
 
 ############################# Heartbeat ######################################
 
-# Configure monitors
+# Define a directory to load monitor definitions from. Definitions take the form
+# of individual yaml files.
+heartbeat.config.monitors:
+  # Directory + glob pattern to search for configuration files
+  path: ${path.config}/monitors.d/*.yml
+  # If enabled, heartbeat will periodically check the config.monitors path for changes
+  reload.enabled: true
```

Comment:
We have this disabled in other Beats by default: https://github.com/elastic/beats/blob/master/filebeat/_meta/common.p2.yml#L64 If I remember correctly the argument was that we do not want to the user to be surprised if we moves files around that it also has directly and affect on his running Beat by default. I wonder if this also applies here?

Sorry, only spotted this now.

Reply:
makes sense, I'll change that.